quodlibet/tests/test_formats__audio.py

```diff
@@ -409,6 +410,193 @@ def test_lyric_filename(self):
         song["lyricist"] = u"Lyricist"
         self.assertTrue(isinstance(song.lyric_filename, fsnative))
 
+    def assert_paths_equal(self, a, b):
+        # pass custom error message to avoid truncation!
+        a2 = os.path.realpath(a).lower()
+        b2 = os.path.realpath(b).lower()
+        msg_fail = str("item1: %s !=\nitem2: %s" % (a2, b2))
```

Comment:
This change was a commit in its own right, `use custom assert to avoid error truncation`, where the *original* comment was much more derogatory - I could not quite believe I was in a situation where the part of the error message (only viewable through one of the build system interfaces) that I needed in order to diagnose the issue, was missing, truncated

However, having now just spent 20 minutes searching through the 100s of build logs for this commit, I can't find the concrete example that drove me to it. Willing to concede that I just couldn't see the wood for the trees at the time ..hence I'll strip it out.

Reply:
I think maybe it depends on the parameter passed to pytest too (or maybe even that's changed over time) e.g. https://docs.pytest.org/en/latest/usage.html#modifying-python-traceback-printing